doc/integration_test_report.rst

```diff
@@ -69,6 +69,7 @@ This text is not part of any item
 .. item:: r005 Another (does not show captions on the related items)
     :aspice: 2
     :asil: C
+    :ext_toolname: namespace:group:another
```

Comment:
```suggestion
    :ext_toolname: namespace
    :group:another
```

Reply:
This wasn't a typo. It's a valid external target, similar to `namespace:group:document` of item `r003`.